src/w3iProxy/index.ts

```diff
@@ -181,23 +187,41 @@ class Web3InboxProxy {
       await this.chatFacade.initInternalProvider(this.chatClient)
     }
 
+    if (this.authProvider === 'internal') {
+      this.authFacade.initInternalProvider()
+    }
+
     if (this.pushProvider === 'internal' && this.uiEnabled.notify && !this.pushClient) {
       this.pushClient = await NotifyClient.init({
         SyncStoreController: SyncStore,
-        identityKeys: this.identityKeys,
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         syncClient: this.syncClient!,
+        identityKeys: this.identityKeys,
         core: this.core
       })
 
       this.pushFacade.initInternalProvider(this.pushClient)
-    }
 
-    if (this.authProvider === 'internal') {
-      this.authFacade.initInternalProvider()
+      if (this.chatClient) {
+        this.chatClient.once('sync_stores_initialized', () => {
+          const account = this.auth.getAccount()
+          if (this.pushClient && account) {
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            const signature = this.chatClient!.syncClient!.signatures.get(
+              `eip155:1:${account}`
+            )!.signature
+
+            this.pushClient.initSyncStores({
```

Comment:
> Allow notify sync stores to be initialized once chat's sync stores are initialized.

Why do we need do make this coupled between the two independent clients here? Basically I'm not following how this relates back to the original issue you described around duplicate spawning of `w3iProxy` and/or silent failure of chat registration.

Reply:
This was the original purpose of this PR before it became a rabbit hole of multiple bug fixes. When you start with a clean slate (`localStorage.clear()`) `this.pushClient.register` never fires because `chatEnabled=true`. 

This mitigates that by not calling `.register` and just initting the sync stores, which is what we care about.